using-mobile-extensions/adobe-journey-optimizer/api-reference.md

```diff
@@ -362,35 +380,35 @@ public String getSound()
 public int getBadgeCount()
 
 // Returns the notification priority from the remote message. 
-// Check out the firebase documentation here (https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages#notificationpriority) 
+// For more information, please read the Firebase documentation (https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages#notificationpriority) 
 public int getNotificationPriority()
 
-// Returns the channel Id from the remote message. 
+// Returns the channel ID from the remote message. 
 public String getChannelId()
 
 // Returns the icon string from the remote message.
 // The icon string represents the drawable resource name in the app.
 public String getIcon()
 
-// Returns the image url from the remote message.
-// The icon string represents the drawable resource name in the app.
+// Returns the image URL from the remote message.
+// The image URL represents the ??? in the app.
```

Comment:
I feel that line 394 can be dropped as "image URL from the remote message" is self explantory.